frontend/src/components/broadcast/Carousel/Carousel.css

```diff
@@ -15,7 +15,7 @@
 }
 
 .slide-card {
-  min-height: 25em;
+  max-height: 26em;
```

Comment:
please make it `min-height: 26em`

Reply:
min-height: 25em
![image](https://user-images.githubusercontent.com/67599130/107947630-9c86f080-6fcd-11eb-8708-83f9fd43ffe7.png)
max-height: 25em
![image](https://user-images.githubusercontent.com/67599130/107947660-a577c200-6fcd-11eb-9ff4-6f2c7c719672.png)
Since the requirements were to reduce the card size I went with max. Do I have to switch back with min-height?

Reply:
I will commit with min-height. If there is a need to change please state thanks.